.github/workflows/test.yml

```diff
@@ -0,0 +1,19 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Test
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm ci
+      - run: npm run format:check
+      - run: npm run build
```

Comment:
@jamesgeorge007 I've added a new test workflow, is this ok?

Reply:
Nevermind, I have removed the build matrix from `master`.